### DISCUSSION TITLE
为什么升级smartdns，磁盘上的永久缓存体积还有缩水的现象呢？我设置的TTL是10年

### DISCUSSION THREAD

**Discussion by yyysuo** [pymumu/smartdns#1437] [Q&A]
按理说所有的域名都不会过期，只会更新域名对应的IP，怎么体积还会变小呢？

bind :8055 -group domestic
bind-tcp :8055 -group domestic
serve-expired-ttl 259200000
serve-expired-prefetch-time 86400
cache-checkpoint-time 36000

**Comment by PikuZheng** [comment]
41.0.22（20230322）及之前的版本，永久缓存和新版不兼容，升级了会清空

**Reply by yyysuo** [reply]
没有缩水到没有，就是从30.2M缩水到30M而已，比较好奇。

**Reply by pymumu** [reply]
ip个数会因测速结果变化
